x-pack/plugins/cross_cluster_replication/public/__jest__/client_integration/follower_indices_list.test.js

```diff
@@ -59,6 +67,54 @@ describe('<FollowerIndicesList />', () => {
     });
   });
 
+  describe('when there are multiple pages of follower indices', () => {
+    let find;
+    let component;
+    let table;
+    let actions;
+    let form;
+
+    const followerIndices = [
+      {
+        name: 'unique',
+        seeds: [],
+      },
+    ];
+
+    for (let i = 0; i < 29; i++) {
+      followerIndices.push({
+        name: `name${i}`,
+        seeds: [],
+      });
+    }
+
+    beforeEach(async () => {
+      httpRequestsMockHelpers.setLoadFollowerIndicesResponse({ indices: followerIndices });
+
+      // Mount the component
+      ({ find, component, table, actions, form } = setup());
+
+      await nextTick(); // Make sure that the http request is fulfilled
```

Comment:
From my notes and finding, we should not add `nextTick()` in our tests anymore but use `jest.useFakeTimers()`